.github/workflows/python-ci.yml

```diff
@@ -95,12 +95,13 @@ jobs:
     - name: Test with coverage
       run: |
         coverage run ./manage.py test -v3
-        coverage report -m
+        coverage xml -o coverage.xml
 
-    - name: Coverage upload
-      run: |
-        pip install codecov
-        codecov
+    - uses: codecov/codecov-action@v3
+      with:
+        files: ./coverage.xml
+        env_vars: OS,ENV
```

Comment:
it makes sense only if you define this env variables with matrix data

https://github.com/GeotrekCE/Geotrek-admin/blob/master/.github/workflows/test.yml#L47

Reply:
define python and django variables and send them to codecov to merge coverages